onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -1593,16 +1593,55 @@ def aten_upsample_bicubic2d_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::upsample_bilinear2d")
```

Comment:
As discussed, we will create a trace_only function that takes all arguments, which conditionally chooses one of the two overloads that handles the `output_size` case and the `scales` case

Reply:
resolved